.github/workflows/lint.yaml

```diff
@@ -0,0 +1,23 @@
+name: lint
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - '**'
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  protobuf:
+    runs-on: ubuntu-latest
```

Comment:
Let's use self-hosted runners.

Also, please rename commit message to "ci: check proto format".